src/main/resources/webapp/interface/castToAllValueTypes.es

```diff
@@ -0,0 +1,193 @@
+// This code is currently not in use, but it useful to see what is possible, and how to do it.
```

Comment:
Why? If it's useful, have it in a readme file somewhere. Or at least clean up the comments.

Reply:
Have code in a readme?

Reply:
This code contains type detection and type casting and might be used soon.

Reply:
Since the file isn't included anywhere it's not even compiled... so it's not wasting cycles